### PR TITLE
Create Modules

### PR DESCRIPTION
**Pull request by Epaltechs**
### In this project, I:
- Divide the Awesome Books code into modules. 
- Save each module in a separate file in the modules folder.
- Import modules in the index.js file. 
- Refactor the methods and functions to arrow functions.
- Display the current date below the navigation bar.

### CODE REVIEW

File: modules/app.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  const date = new Date();
+  const month = date.getMonth();
+  const year = date.getFullYear();
+  const day = date.getDate();
+  time.innerHTML = `${monthNames[month]} ${day} ${year}, ${date.toLocaleTimeString()}`;
+}
+
+setInterval(timer, 1000);
+
+export { timer }
```
Comment (aimemalaika) [inline modules/app.js:17]: - [ ] Kindly user the [Luxon Library](https://github.com/moment/luxon/blob/master/docs/install.md) to get the  [Current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time). Kindly install the package using npm and import it into your project using ES6 syntax as stated in the project requirement.

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Epaltechs, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/app.js (hunk 1 above)
Comment (Epaltechs) [reply modules/app.js:17]: Thank you for the feedback. I downloaded and paste the luxon files into a date.js file and imported it into the index.js file. I had so many linter errors that I could not resolve. Please which approach would you suggest for me to forestall linter errors?

**Comment by Epaltechs** [comment]
### At this point in the project, I:
- Download the luxon script, save it to the project and Import it as a module into index.js file.
- Fix linter errors

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Epaltechs, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**